### ISSUE TITLE
Unable to deserialize/display MsgPack objects

### ISSUE THREAD

**Issue by cbigsby** [redis/RedisDesktopManager#3725]
RDM Version: 0.8.8.384
Environment (OS name and version): Windows 10 x64
Redis Server Version: (multiple) 2.8.2101 Windows

Steps to reproduce:
1. Serialize the following JSON object into MsgPack (in our case using C#'s MsgPack.Cli through StackExchange.Redis.Extensions.MsgPack) and put it into a Redis database:

```
{"ExtendedAttributes":[],"DateAddedUtc":"2015-02-19T15:56:50.117","DateUpdatedUtc":"2015-02-19T15:56:50.117","RmsId":"123","Id":"M5573","Name":"Invalid Catalog Product","MasterProductId":5573,"VariationId":null,"Owner":null,"CanonicalClassification":{"TreeId":1,"Id":56,"Name":"Misc","ParentCategories":[{"Id":12,"Name":"Accessories"},{"Id":44,"Name":"Specialty"}]},"ShortDescription":"This product in a catalog was invalid. It was either missing or it was a private product owned by another company. It was automatically remapped to this product.","LongDescription":"","Manufacturer":null,"MSRP":null,"ReleaseDate":null,"VariationInfo":[],"Specifications":[],"Assets":[],"ColorDefinition":null,"HeroShotUri":null,"HeroShotId":null,"ManufacturerSkus":[],"VendorSkus":[],"UpcCodes":[],"Region":null,"Entity":null,"IsLinkedToCuratedProduct":true,"IsSaleable":true,"IsArchived":false,"Version":1}
```
1. Look up the item in Redis through RDM and try to look at it

Expected result: The item should be shown as MsgPack in the Value page in RDM.

Actual Result: The item is shown as HEX on the Value page. Switching to MsgPack in the "View as:" dropdown results in nothing being shown.

The hex shown is as-follows:

```
\xDC\x00\x1A\x90\x948\xA4Misc\x92\x92\x0C\xABAccessories\x92,\xA9Specialty\x01\xC0\xC0\xC0\xC0\xA5M5573\xC2\xC3\xC3\xA0\xC0\x90\xD1\x15\xC5\xC0\xB7Invalid Catalog Product\xC0\xC0\xC0\xD9\xA1This product in a catalog was invalid. It was either missing or it was a private product owned by another company. It was automatically remapped to this product.\x90\x90\xC0\x90\x90\x01
```

See screenshot: 
![image](https://cloud.githubusercontent.com/assets/812515/18886536/20f7bdae-84a5-11e6-9b8b-9f7e37d40afc.png)
##

**Comment by cbigsby** [comment]
Just a clarification: this happens with all the objects we've serialized with MsgPack and looked at in Redis Desktop Manager. The JSON payload I provided was the smallest object I saw in the small amount of testing I did.

**Comment by uglide** [comment]
@cbigsby Thanks for the feedback
I'll fix it in 0.9 branch

**Comment by uglide** [comment]
Build-in msgpack formatter will be removed and replaced by native formatter - https://github.com/RedisDesktop/rdm-native-value-formatters/issues/6
